proxy/invoke.go

```diff
@@ -133,3 +138,23 @@ func parseHeaders(headers []string) (map[string]string, error) {
 	}
 	return headerMap, nil
 }
+
+// validateMethod validates the HTTP request method
+func validateMethod(method string) (string, error) {
+	var AllowedMethods = []string{http.MethodGet, http.MethodPost}
```

Comment:
Upper-case makes variables public although I don't know how that would behave inside a method, better to name in camelCase.